scripts/ctest/ctest_driver.py

```diff
@@ -13,6 +13,8 @@
 import subprocess
 import sys
 import shutil
+import re
```

Comment:
ctest_driver is not currently used by Jenkins scripts, which directly invoke CTest. I assume this would also be proposing a change to that?

If so, this driver may need to be changed to retain the pattern of executing main-and-smoke in the same CTest invocation (which executes tests across these suites parallel where possible, and groups their results)

Reply:
The proposal is to use the ctest_driver in a separate step after the initial build. While it would be a useful improvement to the script, it's not necessary for this task at the moment.